src/App.js

```diff
@@ -10,7 +10,9 @@ export class App extends Component {
   };
 
   addMovie = (movie) => {
-    // put your code here
+    this.setState(prevState => ({
+      movies: [movie, ...prevState.movies],
```

Comment:
let's add new movie to the end of the array

```suggestion
      movies: [ ...prevState.movies, movie],
```